packages/gatsby-image/src/__tests__/index.js

```diff
@@ -73,7 +73,7 @@ describe(`<Image />`, () => {
     )
     // No Intersection Observer in JSDOM, so placeholder img will be visible (opacity 1) by default
     expect(placeholderImageTag.getAttribute(`style`)).toEqual(
-      `position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 0.5s; color: red;`
+      `position: absolute; top: 0px; left: 0px; width: 100%; height: 100%; object-fit: cover; object-position: center; opacity: 1; transition-delay: 500ms; color: red;`
```

Comment:
Bit of a nit pick! (Feel free to ignore)

Do we need to update the unit from seconds to milliseconds?

Reply:
Usually these kind of animation are under 1s, no ? I found it more readable. I can change it back to seconds :smile: !

Reply:
No, you're right! I'm just wary of change unless absolutely required. This wasn't a public API before this PR so `ms` are good!